src/components/migrationController.js

```diff
@@ -6,6 +6,7 @@ import {updateDB} from '../store/targetDb'
 import ModelTable from './modelTable'
 import RaisedButton from 'material-ui/RaisedButton';
 import AddColumnForm from './addColumnForm'
+import ModelSelector from './modelSelector'
 
```

Comment:
A few of these lines could be refactored using `components/index`
Maybe should be a `Fix` or `Refactor` type PR-- i.e. separate PR?